packages/nextjs-plugin/src/index.ts

```diff
@@ -51,12 +51,22 @@ function withHashicorp({
       ...nextConfig.future,
     }
 
-    const hcPackages = getHashicorpPackages(process.cwd())
+    if (
+      nextConfig.experimental &&
+      (nextConfig.experimental as { transpileModules?: string[] })
+        .transpileModules
+    ) {
+      debugLog(
+        'Disabling next-transpile-modules in favor of experimental transpileModules.'
+      )
```

Comment:
I think we'll still want to pass along the detected hashi packages to the new setting, right?

Reply:
Good call! [`af95c5a` (#112)](https://github.com/hashicorp/web-platform-packages/pull/112/commits/af95c5a72b5001f388aab004f6d47c734ba46ed6)